packages/perseus-linter/src/__tests__/rules.test.ts

```diff
@@ -496,6 +497,25 @@ describe("Individual lint rules tests", () => {
         },
     });
 
+    expectWarning(
+        imageUrlEmptyRule,
+        [
+            "![alt-text]()", // empty URL
+            "![alt-text](  )", // empty URL with spaces
+            "![alt-text](\n)", // empty URL with newline
+        ],
+        null,
```

Comment:
Other similar tests add a ts-expect-error (`// @ts-expect-error - TS2554 - Expected 3 arguments, but got 2.`), but I know those are not ideal, so just added `null` instead. There is no context needed as far as I can tell.

Reply:
That's fine. I think the type for those `expectPass()` and `expectFail()` functions should make `context` be optional.  You could try adding a `?` at the end of `context` and seeing if that works.

Reply:
Wow, that's a beautiful fix! I need to change my mindset from, "Don't change the existing code unrelated to your change" to "Fix the things!" :D Thanks!